lib/game_eval.rb

```diff
@@ -2,20 +2,20 @@ module TicTacToe
 
   class GameEval
 
-    def player_won_or_tied(board, marker1, marker2)
+    def player_won_or_tied(board, player1, player2)
```

Comment:
Can you think of a better name for this method? The name should make it clear that the method will return the winning player